### DISCUSSION TITLE
Feedback/review of my GitHub Profile

### DISCUSSION THREAD

**Discussion by cmulay** [EddieHubCommunity/support#1088] [Show and tell]
Hey there,

Can someone provide me genuine feedback/review of my github [profile](https://github.com/cmulay) ?

@eddiejaoude may be you could look and provide your thoughts / review as well ?

**Comment by schmelto** [comment]
@cmulay maybe you should open an issue for this topic 🤔 

![image](https://user-images.githubusercontent.com/30869493/111297081-86606480-864d-11eb-981e-a8d3814844cb.png)

## Profile

I really like your pinned gists with I am an early... Looks kind of refreshing 👍🏼

**Reply by cmulay** [reply]
Thanks @schmelto ! and sorry I wasn't aware of that issue template, so thought of starting a show/tell discussion.
